src/server/game/Spells/Spell.cpp

```diff
@@ -2295,6 +2295,10 @@ void Spell::DoAllEffectOnTarget(TargetInfo* target)
                 m_caster->ToCreature()->LowerPlayerDamageReq(target->damage);
         }
     }
+	else if (missInfo == SPELL_MISS_IMMUNE2)
+	{
+		m_caster->CombatStart(ObjectAccessor::GetUnit(*m_caster, target->targetGUID)); // Initiate combat with target, despite target being immune
+	}
```

Comment:
tabs2spaces

Reply:
Yes, please convert your 5 tabs to use 4 x space for each indent.